package.json

```diff
@@ -4,7 +4,10 @@
   "description": "",
   "main": "dist/index.js",
   "browser": "dist/index.js",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
So, what about the difference between `peerDependencies` and `optionalDependencies`? Which of them suits better in our case?
https://github.com/aeternity/superhero-button-react/issues/2#issuecomment-656546171

Reply:
optionalDependencies – when package was not installed in some reason other packages will be installed and developer will not see the error.
peerDependencies – when package is required.

Reply:
@davidyuk I think to leave it as it is

Reply:
`peerDependencies` suits better in our case than usual `dependencies` because npm won't allow installing two different versions of `react` (for example) as it may do using usual `dependencies`:

> Trying to install another plugin with a conflicting requirement will cause an error. 

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#peerdependencies

`optionalDependencies` suits better in our case than `peerDependencies` because we don't need npm to warn us about missing `react` package if we are using `superhero-utils` in a project that uses a different UI framework.